topics/persistence.md

```diff
@@ -291,7 +329,10 @@ running. This is what we suggest:
 If you run a Redis instance with only AOF persistence enabled, you can still
 copy the AOF in order to create backups. The file may lack the final part
 but Redis will be still able to load it (see the previous sections about
-truncated AOF files).
+truncated AOF files).  
+
+Since Redis 7.0.0, all the base, incr and manifest files will be placed in a file directory determined by the `appendddirname` configuration.
```

Comment:
```suggestion
Since Redis 7.0.0, all the base, increment and manifest files will be placed in a file directory determined by the `appendddirname` configuration.
```